src/main/java/com/code_intelligence/jazzer/mutation/support/TypeSupport.java

```diff
@@ -186,6 +187,56 @@ public static AnnotatedType notNull(AnnotatedType type) {
     return withExtraAnnotations(type, NOT_NULL);
   }
 
+  /**
+   * Constructs an anonymous WithLength class that can be applied as an annotation to {@code type}
+   * with the given
+   * {@code min} and {@code max} values.
+   * @param type
+   * @param min
+   * @param max
+   * @return {@code type} with a `WithLength` annotation applied to it
+   */
+  public static AnnotatedType withLength(AnnotatedType type, int min, int max) {
```

Comment:
Yeah, that's the unfortunate part of using different annotations. I don't necessarily see a clearly better way of doing this. We could consider allowing `WithUtf8Length` on `byte[]` to simplify this.

Reply:
I don't find this too bad right now. The byte array mutator doesn't care about the distinction between utf8 and not so I think it's conceptually simpler to do this conversion in the string mutator where that distinction is meaningful. We might want to try to simplify how we do annotation manipulation if we find ourselves doing lots of it but that's something we can easily fix later since this isn't part of the public API.